.github/workflows/node.js.yml

```diff
@@ -39,6 +39,9 @@ jobs:
               with:
                 resource: http://localhost:3000
             - run: npm test
+              retry_on: error
+              max_attempts: 2
+              timeout_seconds: 5
```

Comment:
It shows this error when you go into checks:
![error](https://github.com/yaptide/ui/assets/92746092/c7852360-290d-4d2b-822b-f6ef0b46f903)
I dont know why it show that all checks have passed- it looks like the node checks have been skipped in pull request.
According to this: https://www.thisdot.co/blog/how-to-retry-failed-steps-in-github-action-workflows/, I think this fragment should look like this:
```suggestion
            - name: Run tests 
              uses: nick-fields/retry@v2
	      with:
                max_attempts: 2
                retry_on: error
                timeout_seconds: 5
                command: npm test	
```